demo/combo-box-basic-demos.html

```diff
@@ -154,6 +154,33 @@ <h3>Storing custom values</h3>
       </template>
     </vaadin-demo-snippet>
 
+    <h3>Custom overlay width</h3>
+    <vaadin-demo-snippet id="overlay-width" when-defined="vaadin-combo-box">
+      <template preserve-content>
+        <vaadin-combo-box label="Narrow overlay"></vaadin-combo-box>
+        <vaadin-combo-box label="Wide overlay"></vaadin-combo-box>
+        <script>
+          window.addDemoReadyListener('#overlay-width', function(document) {
+            const combos = document.querySelectorAll('vaadin-combo-box');
```

Comment:
Let's add `Array.from` to make sure `forEach` below works in al browsers.

Reply:
I was thinking about that but this demo page already has 2 older demos that also use `forEach()` on the result of `querySelectorAll()` directly and I tested that these demos work in IE11 so I thought it's fine.

But looking at it closer now, it looks like ShadyDOM patches `querySelectorAll()` to return an Array instead of NodeList, so that's why it works.

So I guess it's fine in our demos since they assume ShadyDOM anyway for IE11 and that's the only (still) relevant browser for us that doesn't natively [support Nodelist.forEach](https://caniuse.com/#feat=mdn-api_nodelist_foreach)? Also I don't see how `Array.from` would be better, since that is also not natively supported in IE11, but polyfilled by webcomponentsjs.

Reply:
Imo we only need `Array.from` in this kind of cases when we would use other array methods that are not available directly on [NodeList](https://developer.mozilla.org/en-US/docs/Web/API/NodeList) (or not polyfilled for IE11).

Reply:
Thanks for clarification. No need to change then, let's keep it as it is.